src/features/game/events/landExpansion/plant.ts

```diff
@@ -135,7 +174,13 @@ export function plant({
 
   expansion.plots = plots;
 
-  bumpkin.stamina.value -= PLANT_STAMINA_COST;
+  let staminaCost = PLANT_STAMINA_COST;
+
+  if (bumpkin.skills["Plant Whisperer"]) {
```

Comment:
Similar to the other boosts, should we extract this into a separate function for readability + testibility?